### PR TITLE
chore/improve sdk

### PR DESCRIPTION
**Pull request by maycon-mello**
Blocked by https://github.com/docknetwork/react-native-sdk/pull/23

### CODE REVIEW

**Comment by mike-parkhill** [comment]
https://github.com/docknetwork/react-native-sdk/pull/23 is merged, but we're still failing

File: src/features/wallet/wallet-slice.test.js [hunk 1]
```diff
@@ -36,43 +34,43 @@ describe('Wallet Slice', () => {
...
-    walletService.query.mockResolvedValue(JSON.parse(docs));
-    return importWallet({}, 'Password');
-  });
+  // it('expect to  import wallet with accurate data', () => {
```
Comment (mike-parkhill) [inline src/features/wallet/wallet-slice.test.js:37]: what's the plan for resurrecting this test?

File: src/features/transactions/transactions-slice.test.js [hunk 2]
```diff
@@ -48,37 +40,4 @@ describe('transactions-slice', () => {
     expect(sorted[2]).toBe(t2);
     expect(sorted[3]).toBe(t1);
   });
-
```
Comment (mike-parkhill) [inline src/features/transactions/transactions-slice.test.js:51]: why don't we need this test anymore?

Comment (maycon-mello) [reply src/features/transactions/transactions-slice.test.js:51]: This is testing realm db from the app side, which is now handled by the wallet-sdk

File: src/features/wallet/wallet-slice.test.js (hunk 1 above)
Comment (maycon-mello) [reply src/features/wallet/wallet-slice.test.js:37]: I think it should be handled by the wallet-sdk as well. Will just check maybe would need to move this to the other repo

File: src/features/tokens/SendTokenScreen.js [hunk 3]
```diff
@@ -203,8 +202,13 @@ const defaultFormState = {
 
 export function SendTokenContainer({route}) {
   const dispatch = useDispatch();
-  const {address, recipientAddress = ''} = route.params || {};
-  const accountDetails = useSelector(accountSelectors.getAccountById(address));
+  const {
+    address,
+    recipientAddress = '3B55QmdMTBo7SsWfFsmbVJCL1LkRdMnno3PCScwgiT4Npcso',
```
Comment (mike-parkhill) [inline src/features/tokens/SendTokenScreen.js:207]: trying to get some bonus tokens sent your way? 😛

Comment (maycon-mello) [reply src/features/tokens/SendTokenScreen.js]: 🏃🏻💰

File: src/features/wallet/wallet-slice.test.js (hunk 1 above)
Comment (mike-parkhill) [reply src/features/wallet/wallet-slice.test.js:37]: what's the verdict? Does the test get moved, or just deleted from here?

**Pull request merged**